docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -37,8 +37,9 @@ The following tokens are issued:
 
 See also the xref:security-oidc-configuration-properties-reference.adoc[OIDC configuration properties] reference guide.
 
-For information about protecting your applications using Bearer Token authorization, see xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer authentication].
+To learn about how you can protect web applications by using the OIDC authorization code flow mechanism, see xref:security-oidc-code-flow-authentication-tutorial.adoc[Protect a web application by using OIDC authorization code flow]
 
+If you want to protect your applications by using Bearer Token authorization, see xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer authentication].
```

Comment:
```suggestion
If you want to protect your applications by using Bearer Token authentication, see xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer authentication].
```

Reply:
`authorization` was there earlier, but `authentication` would be better, the concept also has `authentication`